### PR TITLE
server: add get first region meta support by using empty search key.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
@siddontang  @disksing PTAL

### CODE REVIEW

File: server/cluster_worker_test.go [hunk 1]
```diff
@@ -732,7 +732,14 @@ func (s *testClusterWorkerSuite) TestSplit(c *C) {
...
 
-	for _, t := range tbl {
+	var firstRegionStartKey, firstRegionEndKey string
+	for i, t := range tbl {
+		// Get first region start and end key.
+		if i == 0 {
+			firstRegionStartKey = t.startKey
+			firstRegionEndKey = t.splitKey
+		}
+
```
Comment (disksing) [inline server/cluster_worker_test.go]: I think `firstRegionStartKey, firstRegionEndKey := tbl[0].startKey, tbl[0].splitKey` would be more straight forward.

Comment (qiuyesuifeng) [inline server/cluster_worker_test.go]: Maybe better, actually i have done as what u want.

**Comment by siddontang** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
@disksing PTAL

**Comment by disksing** [comment]
LGTM

**Pull request merged**